### ISSUE TITLE
Stay open in down mode

### ISSUE THREAD

**Issue by M5oul** [ucoin-io/cutecoin#190]
It could be great if QtCoin could stay running and closed in bottom left corner. This is a GNOME feature. I don't know if other desktop environnements hold that kind of feature.
![gnome_down app](https://cloud.githubusercontent.com/assets/4758871/9339885/cf515ce6-45ec-11e5-80a5-2d1306b4466f.png)
Here you could see apps I just need to open sometimes.

**Comment by Insoleet** [comment]
Yeah, I always thought it'd be great. But not before some time I guess.
